detectors/node/opentelemetry-resource-detector-instana/src/detectors/InstanaAgentDetector.ts

```diff
@@ -13,31 +13,44 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-import { Detector, Resource, IResource } from '@opentelemetry/resources';
+import {
+  DetectorSync,
+  Resource,
+  IResource,
+  ResourceAttributes,
+} from '@opentelemetry/resources';
 import { diag } from '@opentelemetry/api';
 import {
   SEMRESATTRS_PROCESS_PID,
   SEMRESATTRS_SERVICE_INSTANCE_ID,
 } from '@opentelemetry/semantic-conventions';
 import * as http from 'http';
 
-class InstanaAgentDetector implements Detector {
+class InstanaAgentDetector implements DetectorSync {
```

Comment:
While very unlikely, there might be users out there which can potentially break due to this change. 
Is there any reason to avoid marking it with `feat!` just in case?